packages/access-client/src/agent.js

```diff
@@ -476,30 +478,6 @@ export class Agent {
     }
   }
 
-  /**
-   * Request authorization of a session allowing this agent to issue UCANs
-   * signed by the passed email address.
-   *
-   * @param {`${string}@${string}`} email
-   * @param {object} opts
-   * @param {AbortSignal} [opts.signal]
-   * @param {Iterable<{ can: Ucanto.Ability }>} opts.capabilities
-   */
-  async requestAuthorization(email, opts) {
```

Comment:
note: removing `requestAuthorization` method in favor of a [use case](https://blog.cleancoder.com/uncle-bob/2012/08/13/the-clean-architecture.html#use-cases) called `requestAuthorization` that operates on an `AccessClient`, but doesn't require adding a new method to `AccessClient` interface